src/logging/request-obfuscator.js

```diff
@@ -31,6 +31,10 @@ function obfuscateResponseBody(body, method) {
  */
 
 function obfuscateResponse(request, instance) {
+  if (request.type !== 'response') {
```

Comment:
Maybe create a separate commits for this and the tests?

Reply:
Done.